log-insights/setup/aiven/03_configure_collector.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Step 3: Configure the Collector'
+install_track_title: 'Log Insights: Setup (Aiven)'
+backlink_href: /docs/log-insights/setup
+backlink_title: 'Log Insights: Setup'
+---
+
+import imgGcpFirewallRules from './gcp-firewall-rules.png'
+import imgAwsSecurityGroup from './aws-security-group.png'
+
+export const ImgGcpFirewallRules = () => <img alt="GCP firewall rules allowing access on syslog port" src={imgGcpFirewallRules} />
+
+export const ImgAwsSecurityGroup = () => <img alt="AWS Security Group allowing access on syslog port" src={imgAwsSecurityGroup} />
+
+We can now configure the syslog receiver in the collector. Edit the collector
+config and add the
+[`db_log_syslog_server`](https://pganalyze.com/docs/collector/settings#self-managed-servers)
+setting. This should be set to a local address on any free port:
+
+```
+db_log_syslog_server: 0.0.0.0:32514
+```
+
+(If using Docker, use the `LOG_SYSLOG_SERVER` environment variable instead.)
+
+It's recommended to use an unprivileged port number (1024 and up) to avoid
+running the collector as root.
+
+Restart the collector.
+
+Make sure that the port you've selected is not blocked by a firewall or security group rules.
+E.g., in GCP, you'll need to add a firewall rule:
+
+<ImgGcpFirewallRules />
+
+In AWS, you'll need to add a security group rule:
+
+<ImgAwsSecurityGroup />
+
+You can verify that the port is accessible with telnet:
+
+```
+$ telnet 203.0.113.2 50014
```

Comment:
Since we talk about the VPC peering and keeping things private earlier, its a bit odd that we are now referencing a public IP here (and a specific IP as well?)

maybe we should clarify which IP we mean here? (presumably the public IP of where the collector runs)

Reply:
That's a great point. I think this was from a note I made when I was debugging and was just trying to verify that a public internet connection worked. But if we want to encourage a secure setup, we don't want this to be accessible over the public internet at all (no one can snoop on your log traffic if that connection is using the VPC peering, but you could still get a DoS or spam with a setup like that). Ideally, we'd want you to verify that you can connect from the Aiven VPC , but you can't really run `nc` there. Maybe we should just drop this altogether?

The IP I chose is from https://datatracker.ietf.org/doc/html/rfc5737, but yeah maybe that's surprising if you don't know about it.

Reply:
Yeah, maybe dropping it is fine - this isn't really specific to our setup (though helpful to test security group settings).